apps/web/src/app/(docs)/docs/sandbox/persistence/page.mdx

```diff
@@ -128,6 +128,84 @@ print('Sandbox resumed', same_sbx.sandbox_id) # $HighlightLine
 ```
 </CodeGroup>
 
+## 4. Listing paused sandboxes
+You can list all paused sandboxes by calling the `Sandbox.list` method by supplying the `state` parameter.
+
+<CodeGroup>
+```js
+import { Sandbox } from '@e2b/code-interpreter'
+// or use Core: https://github.com/e2b-dev/e2b
+// import { Sandbox } from 'e2b'
+//
+// or use Desktop: https://github.com/e2b-dev/desktop
+// import { Sandbox } from '@e2b/desktop'
+
+// List all paused sandboxes
+const sandboxes = await Sandbox.list({ state: 'paused' }) // $HighlightLine
+console.log('Paused sandboxes', sandboxes) // $HighlightLine
+```
+```python
+from e2b import Sandbox
+# or use Core: https://github.com/e2b-dev/e2b
+# from e2b import Sandbox
+#
+# or use Desktop: https://github.com/e2b-dev/desktop
+# from e2b_desktop import Sandbox
+
+# List all paused sandboxes
+sandboxes = Sandbox.list(state='paused') # $HighlightLine
+print('Paused sandboxes', sandboxes) # $HighlightLine
+```
+</CodeGroup>
+
+## 5. Removing paused sandboxes
```

Comment:
why is js and python little bit different, also could you unify word `id`, sometimes you have it in lowercase and it other instance in uppercase

Reply:
Can you give me an example?